src/python/lib/clusterfuzz/fuzz/__init__.py

```diff
@@ -19,10 +19,17 @@
 
 
 def _initialize():
+  """Initialize the engine implementations."""
   global _initialized
 
```

Comment:
Maybe add a comment next to three 3 try-except that this is temporary until internal clusterfuzz dependencies are untangled from the python package.

Reply:
Realistically I'm not sure internal clusterfuzz dependencies can be completely removed, there are just too many things that e.g. libFuzzer engine implementation depends on. The best we can do is to hide them, as I've tried to do here.